src/containers/Article/index.js

```diff
@@ -61,6 +65,14 @@ export class ArticleForm extends Component {
     this.setState({ body: blocks });
   };
 
+  handleTags = (newValue) => {
+    const tags = [];
+    newValue.map((value) => {
```

Comment:
same case here, the name `newValue` is singular, but again you are mapping over it and expecting a value!!!! if its an array, the name should be pluralized

Reply:
On it